src/LVSS.cpp

```diff
@@ -17,7 +14,15 @@ CO_OBJ_T* LVSS::getObjectDictionary() {
     return nullptr;
 }
 
-void LVSS::process() {
+LVSS::LVSS(TPS2HB50BQ1* powerSwitches[POWER_SWITCHES_SIZE]) {
```

Comment:
There's an example of how to do this with an initializer list in [here](https://github.com/RIT-EVT/TMS/blob/main/src/TMS/dev/TCA9545A.cpp). It's not particularly neat, but it gets the job done.